cloud/scaling-a-service.md

```diff
@@ -101,7 +80,13 @@ very quickly. However, if you have a lot of existing data, optimization can take
 longer, and in some cases this could create a delay longer than six hours.
 
 Autoscaling can only increase disk size, not decrease it. You can have a disk up
-to 10&nbsp;TB in size.
+to 16&nbsp;TB in size.
+
+If you have a Timescale Cloud multi-node environment, you can also use
+autoscaling. We recommend that you define different scale limits for the access
+node and data nodes, not just because they have different workloads, but also
+because access nodes should scale much more slowly than data nodes. Data nodes
```

Comment:
When we are talking that they are going to scale more slowly, it makes an impression that the actual scaling operation for them is going to last longer.

```suggestion
because access nodes are less demanding for storage than data nodes. Data nodes
```